streams/src/test/java/org/apache/kafka/streams/state/internals/ListValueStoreTest.java

```diff
@@ -178,26 +181,46 @@ public void shouldGetAllReturnTimestampOrderedRecords() {
 
     @Test
     public void shouldAllowDeleteWhileIterateRecords() {
-        listStore.put(0, "zero1");
-        listStore.put(0, "zero2");
-        listStore.put(1, "one");
+        final Random rand = new Random();
```

Comment:
Should we get a `seed` value and log it (to allow reproducing if the test fails)?
```
final long seed = new Random(System.currentTimeMs).nextLong();
log(seed);
final Random rand = new Random(seed);
```